### PR TITLE
add token for table item.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @superyyrrzz @hellosnow @ansyral @qinezh

### CODE REVIEW

**Comment by ansyral** [comment]
:shipit:

**Comment by qinezh** [comment]
:+1:

File: src/Microsoft.DocAsCode.MarkdownLite/Tables/MarkdownTableItemBlockToken.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.MarkdownLite/Tables/MarkdownTableItemBlockToken.cs]: remove extra line

File: src/Microsoft.DocAsCode.MarkdownLite/Tables/MarkdownTableItemBlockToken.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+namespace Microsoft.DocAsCode.MarkdownLite
+{
+    public class MarkdownTableItemBlockToken : IMarkdownToken, IMarkdownRewritable<MarkdownTableItemBlockToken>
+    {
+        public MarkdownTableItemBlockToken(
+            IMarkdownRule rule,
+            IMarkdownContext context,
+            InlineContent content,
+            SourceInfo sourceInfo)
+
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.MarkdownLite/Tables/MarkdownTableItemBlockToken.cs]: remove extra line?

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**